contrib/mod_tls.c

```diff
@@ -11984,7 +11984,10 @@ MODRET set_tlsciphersuite(cmd_rec *cmd) {
   c = add_config_param(cmd->argv[0], 1, NULL);
 
   /* Make sure that EXPORT ciphers cannot be used, per Bug#4163. */
-  c->argv[0] = pstrcat(c->pool, "!EXPORT:", ciphersuite, NULL);
+  /* This breaks system profiles though, so don't change them.   */
+  if (strncmp(ciphersuite, "PROFILE=", 8)) {
+    c->argv[0] = pstrcat(c->pool, "!EXPORT:", ciphersuite, NULL);
+  }
```

Comment:
See https://github.com/proftpd/proftpd/pull/500; may merge the `PROFILE` handling PR is after that one?